jboss-kie-workbench/added/launch/jboss-kie-workbench.sh

```diff
@@ -129,6 +130,37 @@ function configure_guvnor_settings() {
             JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.uberfire.nio.git.hooks=${GIT_HOOKS_DIR}"
         fi
     fi
+
+    local hostname
+    local port="80"
+    local protocol="http"
+    if [ "${WORKBENCH_ROUTE_NAME}" != "" ]; then
+
+        if [[ "${WORKBENCH_ROUTE_NAME}" = *"secure"* ]]; then
+            port="443"
+            protocol="https"
+        fi
+
+        local response=$(query_server_host ${WORKBENCH_ROUTE_NAME})
+
+        if [ "${response: -3}" = "200" ]; then
+               # parse the json response to get the route host
+               url=$(echo ${response::- 3} | python -c 'import json,sys;obj=json.load(sys.stdin);print obj["spec"]["host"]')
+               hostname="${protocol}://${url}:${port}"
```

Comment:
Sorry to be picky, but it looks like the variable names are reversed. $url is actually the hostname, and $hostname is actually the url in this code. Can you switch them please? It would make the code more readable and maintainable. Thank you!